### PR TITLE
add grid layout for events

### PR DESCRIPTION
**Pull request by ac1214**

### CODE REVIEW

File: src/components/EventCard/EventCard.tsx [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
-const EventCard = () => {
-  const events = useSelector((state: ReduxState) => {
-    return state.events.events.getAllData();
-  });
-
+const EventCard = (props: any) => {
   const classes = eventCardStyles();
-
   return (
+<<<<<<< HEAD
```
Comment (Brian-Pho) [inline src/components/EventCard/EventCard.tsx]: This shouldn't be here

Comment (ac1214) [reply src/components/EventCard/EventCard.tsx]: lol removed it

File: src/components/EventGrid/EventGrid.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const EventGrid = (props: any) => {
+  const events = useSelector((state: ReduxState) => {
+    return state.events.events.getAllData();
+  });
+
+  return (
+    <Container maxWidth="xl">
+      <Grid container spacing={3}>
+        {events.map((event, index) => {
```
Comment (Brian-Pho) [inline src/components/EventGrid/EventGrid.tsx]: Index isn't being used yet. Use it as the key attribute for the EventCard.

File: src/components/EventCard/EventCard.tsx (hunk 1 above)
Comment (ac1214) [reply src/components/EventCard/EventCard.tsx]: Forgot to save the file before commit

**Pull request merged**